src/qibolab/instruments/simulator/models/methods.py

```diff
@@ -0,0 +1,40 @@
+from typing import Union
+
+import yaml
+
+MHz = 1e6
+GHz = 1e9
+us = 1e-6
+ns = 1e-9
```

Comment:
Why do you need all these conversions?

`MHz` and `us` are never used, and you could save the other two by expressing everything in GHz and ns from the beginning (I believe you will never need to use times of ~1s).

Reply:
Btw, in these definitions, `Hz` and `s` are superfluous: those the definition of the prefixes themselves...

Reply:
Yes it is true these are not necessary and originally I didn't have these but I saw it in Paul's code and thought it was pretty useful visually to see the units in the code when expressing values of frequency/time and avoids typos in the exponent, so e.g. `2.0*GHz` and `0.5*ns` as opposed to `2.0*1e9` and `0.5/1e9`. It's mostly just GHz and ns anyway so yes I can remove the other 2.

These conversion units are used for converting the values from `model.json` to GHz/ns which enters the simulation. Yes in fact I originally had the model parameters directly expressed in GHz and ns, but I thought it would be better to have consistent units with the runcard to avoid confusion.

Reply:
> Yes it is true these are not necessary and originally I didn't have these but I saw it in Paul's code and thought it was pretty useful visually to see the units in the code when expressing values of frequency/time and avoids typos in the exponent, so e.g. `2.0*GHz` and `0.5*ns` as opposed to `2.0*1e9` and `0.5/1e9`. It's mostly just GHz and ns anyway so yes I can remove the other 2. Also it is just an added option for model developers, they can choose to use it if they find it more helpful, or just use 1e9.

My point was mainly to avoid both the GHz and the `1e9`, and just assume it is always GHz (keeping the conversion implicit, and doing it at the innermost level, i.e. within QuTip... or not even that, if the physics involved is invariant under time dilation)

Reply:
So this conversion has to take place somewhere mainly because the user inputs values in Hz in the model.json, but it will be more efficient to do the simulation in GHz. Do you mean that you prefer users to input parameters in units of GHz in model.json (i.e. different from runcard convention) to avoid the conversion, over having the same convention with the runcard?

Reply:
> Do you mean that you prefer users to input parameters in units of GHz in model.json

Exactly this. Is there any other reason to input the value in Hz?

Reply:
The main and only reason is to be consistent with the runcards actually.

Reply:
Actually, I'd like to change it even there...

Reply:
Perfect, I will switch to implicit GHz units then.

Reply:
Fixed in 490e7fb